video/vdu_audio.h

```diff
@@ -0,0 +1,321 @@
+//
```

Comment:
to aid readability/maintainability I split the audio VDU commands out into a separate file